src/Cli/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -531,6 +531,9 @@ Run 'dotnet {1} --show-aliases' with no args to show all aliases.</value>
   <data name="TemplatePackageCoordinator_Install_Error_FoundNoPackagesToInstall" xml:space="preserve">
     <value>Found no template packages to install.</value>
   </data>
+  <data name="Colon_Separator_Deprecated" xml:space="preserve">
+    <value>The colon separator "::" has been deprecated in favor of the at symbol "@" for separating the package from the version in dotnet new install. In your case, this means {0}@{1} instead of {0}::{1}.</value>
```

Comment:
I think it is more commonly referred to as the `at sign`. Idk why it is done that way, but best stick with conventions most people would easily recognize. Also, I think `dotnet new install` should be wrapped in single quotes and I believe there is a way in the resx to let the translators know that `dotnet new install` should not be translated.